melodic/distribution.yaml

```diff
@@ -15510,6 +15510,24 @@ repositories:
       url: https://github.com/ElettraSciComp/witmotion_IMU_ros.git
       version: main
     status: maintained
+  witmotion_ros-release:
```

Comment:
The name of the key should not contain `-release`; in this case, it should just be `witmotion_ros`.

(the fact that it is this way in noetic is a bug that should be fixed)

Reply:
Fixed with [59a67b7](https://github.com/ros/rosdistro/pull/36427/commits/59a67b778cc2bd5c4fabd87dfb15afe9d7273eb3)